test/plugin_functional/plugins/core_plugin_a/public/application.tsx

```diff
@@ -95,7 +95,7 @@ const Nav = withRouter(({ history, navigateToApp }: NavProps) => (
           {
             id: 'home',
             name: 'Home',
-            onClick: () => history.push('/'),
+            onClick: () => history.push(''),
```

Comment:
`prependBasePathToString` was incorrectly removing the trailing `/`, as this is fixed, I adapted the test plugins instead or changing every `waitForUrlToBe('/app/foo');` to `waitForUrlToBe('/app/foo/');` in `test/plugin_functional/test_suites/core_plugins/applications.ts`